Docker/ods-api-web-api/alpine/pgsql/run.sh

```diff
@@ -25,7 +25,7 @@ do
       PGHOST=$HOST \
       PGPORT=$POSTGRES_PORT \
       PGUSER=$POSTGRES_USER \
-      pg_isready > /dev/null
+      pg_isready -h "$HOST" -p "$POSTGRES_PORT" -U "$POSTGRES_USER" -d "$POSTGRES_DB" > /dev/null
```

Comment:
No need to specify the host, port and user since they already come from the `PGHOST`, `PGPORT`, `PGUSER` env vars respectively (a couple of lines above). So this line can be changed to:
`pg_isready -d "$POSTGRES_DB" > /dev/null`